src/analyzer.rs

```diff
@@ -145,36 +145,41 @@ pub fn process_directory(args: &Cli) -> Result<()> {
 
 fn is_excluded(entry: &ignore::DirEntry, args: &Cli) -> bool {
     if let Some(excludes) = &args.exclude {
-        let path_str = entry.path().to_string_lossy();
-
+        let path = entry.path();
         for exclude in excludes {
             match exclude {
                 Exclude::Pattern(pattern) => {
-                    let pattern = if !pattern.starts_with("./") && !pattern.starts_with("/") {
-                        format!("./{}", pattern)
```

Comment:
This was pretty much needed because `ignore::DirEntry` was using absolute paths. and if i do things like `-e "src/*"` it does not work without it.